dbt_semantic_interfaces/dbt_semantic_interfaces/dataclass_serialization.py

```diff
@@ -18,12 +18,10 @@
     get_origin,
     get_type_hints,
 )
-from typing_extensions import TypeAlias
 
 import pydantic
 from pydantic import BaseModel
-
-from dbt_semantic_interfaces.pretty_print import pformat_big_objects
```

Comment:
This commit message is now very confusing, but ok. :)

Reply:
Yeah, this commit is no longer needed since `pformat_big_objects` got moved from `metricflow` to `dbt_semantic_interfaces`.